test/e2e/21_ssh_key_generation.bats

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bats
+
+function setup() {
+  load lib/env
+  load lib/install
+  load lib/poll
+  load lib/defer
+
+  kubectl create namespace "$FLUX_NAMESPACE"
+
+  install_flux_with_fluxctl
+}
+
+@test "SSH key is generated" {
+  run fluxctl identity --k8s-fwd-ns "$FLUX_NAMESPACE"
+  [ "$status" -eq 0 ]
+  [ "$output" != "" ]
```

Comment:
I think we can do a bit better than this. Could parse the output and check it's a valid public key?

Reply:
Also, we may want to do polling, since the key may not be ready right after flux is deployed.

Reply:
The key is now validated in two ways:

1. it is parsed by `ssh-keygen` to compute the fingerprint, which will fail for an invalid key
2. the fingerprint is compared to the output of `fluxctl identity -l`